tests/unit/tournament_pair_info.spec.ts

```diff
@@ -1,5 +1,6 @@
-import { pairBoardCount, pairWinnerCount, pairTableCount } from "@/composables/tournamentPairInfo";
+// 
```

Comment:
```suggestion
// This program has been developed by students from the bachelor Computer Science at 
// Utrecht University within the Software Project course.

// © Copyright Utrecht University (Department of Information and Computing Sciences)
```

of hoort er hier geen?